fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1619,6 +1619,28 @@ private TNetworkAddress toBrpcHost(TNetworkAddress host) throws Exception {
         return new TNetworkAddress(computeNode.getHost(), computeNode.getBrpcPort());
     }
 
+    private String backendInfosString(boolean chooseComputeNode) {
+        if (chooseComputeNode) {
+            String infoStr = "compute node: ";
+            for (Map.Entry<Long, ComputeNode> entry : this.idToComputeNode.entrySet()) {
+                Long backendID = entry.getKey();
+                ComputeNode backend = entry.getValue();
+                infoStr += String.format("[%d-%s-%d alive: %b inBlacklist: %b] ", backendID, backend.getHost(), 
+                                    backend.getBePort(), backend.isAlive(), SimpleScheduler.isInBlacklist(backendID));
+            }
+            return infoStr;
+        } else {
+            String infoStr = "backend: ";
+            for (Map.Entry<Long, Backend> entry : this.idToBackend.entrySet()) {
+                Long backendID = entry.getKey();
+                Backend backend = entry.getValue();
+                infoStr += String.format("[%d-%s-%d alive: %b inBlacklist: %b] ", backendID, backend.getHost(), 
```

Comment:
IP is enough. The backendID and port is not necessary

Reply:
```
backend: xxx. error: xxx
```
is clear

Reply:
done